data/sql/updates/pending_db_world/rev_1612957477367642592.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1612957477367642592');
+
+-- Lower respawn of and add more spawn locations for Tirisfal Pumpkin
+￼
```

Comment:
remove weird symbol

Reply:
Dammit, seems like some invisible stuff is added when I copy/paste stuff from the browser (I based this on a previous PR)